### ISSUE TITLE
[distributed] control: Implement a way to clear stale locks

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio#2898]
Current 'control lock' cli doesn't have a way to clear stale locks.

**Comment by harshavardhana** [comment]
// cc @krishnasrinivas

**Comment by GarimaKapoor** [comment]
@fwessels can you take a look at this.

**Comment by fwessels** [comment]
@GarimaKapoor am working on a dsync branch for this

**Comment by GarimaKapoor** [comment]
great! @fwessels

**Comment by fwessels** [comment]
See https://github.com/minio/minio/pull/3013

Structure of control lock command is now as follows:

```
  1. List all currently active locks from all nodes. Defaults to list locks held longer than 24hrs.
    $ minio control lock list http://localhost:9000/

  2. List all currently active locks from all nodes. Request locks older than 1minute.
    $ minio control lock --older-than=1m list http://localhost:9000/

  3. Clear lock named 'bucket/object' (exact match).
    $ minio control lock clear http://localhost:9000/bucket/object

  4. Clear all locks with names that start with 'bucket/prefix' (wildcard match).
    $ minio control lock --recursive clear http://localhost:9000/bucket/prefix

  5. Clear all locks older than 10minutes.
    $ minio control lock --older-than=10m clear http://localhost:9000/

  6. Clear all locks with names that start with 'bucket/a' and that are older than 1hour.
    $ minio control lock --recursive --older-than=1h clear http://localhost:9000/bucket/a
```

**Comment by harshavardhana** [comment]
Fixed by #3013
